lsusb.c

```diff
@@ -2200,29 +2200,34 @@ static void dump_ccid_device(const unsigned char *buf)
 		fputs("          Auto clock change\n", stdout);
 	if ((us & 0x0020))
 		fputs("          Auto baud rate change\n", stdout);
-	if ((us & 0x0040))
+	if ((us & (0x0040 | 0x0080)) == 0x0040)
 		fputs("          Auto parameter negotiation made by CCID\n", stdout);
-	else if ((us & 0x0080))
+	else if ((us & (0x0040 | 0x0080)) == 0x0080)
 		fputs("          Auto PPS made by CCID\n", stdout);
 	else if ((us & (0x0040 | 0x0080)))
 		fputs("        WARNING: conflicting negotiation features\n", stdout);
 
 	if ((us & 0x0100))
 		fputs("          CCID can set ICC in clock stop mode\n", stdout);
 	if ((us & 0x0200))
-		fputs("          NAD value other than 0x00 accepted\n", stdout);
+		fputs("          NAD value other than 0x00 accepted (T=1)\n", stdout);
```

Comment:
Please explain somewhere what "T=1" means.

Reply:
See iso7816-3. There are 2 standardized options for the low level communication protocol, one character-based protocol (T=0) and one block-based protocol (T=1). The NAD (node-address) communication parameter is only relevant for T=1. People who are familiar with NAD are also familiar with T=1.

Reply:
thanks!